ui/src/components/data-agent/SyncInitiated.tsx

```
@@ -0,0 +1,61 @@
+import { FC } from 'react';
+
+interface SyncInitiatedProps {}
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)